apps/client/src/features/rundown/event-block/EventBlockInner.tsx

```diff
@@ -10,6 +10,7 @@ import { IoPlayForward } from '@react-icons/all-files/io5/IoPlayForward';
 import { IoPlaySkipForward } from '@react-icons/all-files/io5/IoPlaySkipForward';
 import { IoStop } from '@react-icons/all-files/io5/IoStop';
 import { IoTime } from '@react-icons/all-files/io5/IoTime';
+import { MdInput } from '@react-icons/all-files/md/MdInput';
```

Comment:
I would prefer if we stick to Io5.

Either that or find a new icon set which has everything we need

Reply:
agree, it is just a quick place holder

Reply:
IoFlash 
![image](https://github.com/cpvalente/ontime/assets/19875125/d077b457-028e-418f-b3f9-cad03f9c8835)

IoArrowForwardCircleOutline 
![image](https://github.com/cpvalente/ontime/assets/19875125/c1f1ee83-e85c-4ba6-a590-982c692d44a6)

IoDiscOutline 
![image](https://github.com/cpvalente/ontime/assets/19875125/c658bc89-6e04-4dae-bfec-039a121f0b46)

MdOutlineSettingsInputSvideo 
![image](https://github.com/cpvalente/ontime/assets/19875125/dd74222b-52ae-4c54-94c2-e2397ce71533)